src/lamplib/src/genny/cli.py

```diff
@@ -212,12 +212,25 @@ def clean(ctx: click.Context) -> None:
 
 
 @cli.command(name="cmake-test", help="Run genny's C++ unit tests.")
+@click.option(
+    "-g", "--regex", required=False, default=None, help=("Regex to match against tests."),
+)
+@click.option(
+    "-r",
+    "--repeat-until-fail",
+    required=False,
+    default=1,
+    help=("Repeat each test this many times until failing. Default is 1."),
```

Comment:
This was hard for me to parse mentally. Really the flag is "repeat this many times unless it fails." Allows for tests to be flaky and dominated by passing or false-misses (didn't catch cuz only ran once and it only fails small-n% of the time). Not suggesting a change but I got confused by it, so perhaps an explanation of when/why you'd want to use this would be helpful.